### ISSUE TITLE
Give users more freedom while validating donations

### ISSUE THREAD

**Issue by sivaraam** [mozilla/voice-web#407]
It seem that the web interface doesn't give the user with much freedom with the voice they are validating. It would be nice to have a way to **skip** the current validation as the user may not be sure enough to decide whether it's correct or not. The current interface only asks Yes / Nope which could motivate users to give incorrect feedback just to *skip* the current validation.

BTW, the options seem to need some improvement in wordings too. Yes / Nope doesn't seem to be a good pair of words for the given context. As there's no textual gesture related to the question, how about no wordings at all for the options too! i.e., only the tick and the cross.

**Comment by azymohliad** [comment]
As a quick hack I just refresh page, when I want to skip.

**Comment by mikehenrty** [comment]
There are two separate bugs here:
1. lack of ability to skip recordings
1. lack of explanation of what "Yes!" and "Nope."

Let's use this bug to track the first one. @sivaraam would you mind filing a new bug for number 2, since we could land that in a separate commit?

**Comment by Omniscimus** [comment]
Bug number 2 might already be covered by #273

**Comment by sivaraam** [comment]
Yeah, I guess @Omniscimus's correct. There seems to be quite a lot of background there. #284 seems to cover number 2 more correctly.

**Comment by mikehenrty** [comment]
see also #278, which is about skipping speak

**Comment by SimonHFrost** [comment]
I could take up this task if nobody else is already looking at it. 

I would add a skip button similar to in the screenshot here:

![screen shot 2017-12-07 at 21 06 20](https://user-images.githubusercontent.com/877992/33736610-89f82a64-db93-11e7-93ff-ca5f82f11967.png)

Once you reach the confirmation, the sentence you’ve skipped would simply not appear in the list, and not get sent to the backend when confirming.

There would be an edge case when a user skips all three sentences, then I’d show perhaps a new button that says ‘Get new sentences’, and navigate back.

Does that sound good?

**Comment by SimonHFrost** [comment]
I'd also recommend closing https://github.com/mozilla/voice-web/issues/278 as a duplicate (there's more info in this issue)

**Comment by Gregoor** [comment]
Hi @SimonHFrost, this issue is actually about skipping sentences when validating (the widget on the home page). Let's discuss those ideas at #278, which is about skipping when recording
